packit_service/service/events.py

```diff
@@ -465,7 +499,7 @@ def __init__(
             self.base_repo_namespace = build.get("repo_namespace")
             https_url = build["https_url"]
 
-        super().__init__(trigger=JobTriggerType.commit, project_url=https_url)
+        super().__init__(trigger=JobTriggerType.pull_request, project_url=https_url)
```

Comment:
This is the problem I want to discuss on the arch meeting:
- We need to differentiate between jobs and the `part of the config which contains the metadata about the job`.
- We need to support mapping from jobs to the `part of the config which contains the metadata about the job`.